circuits/plonk-15-wires/src/lookup_spec.rs

```diff
@@ -0,0 +1,211 @@
+use ark_ff::{Field, FftField};
+use std::collections::{HashMap, HashSet};
+use ark_poly::{Radix2EvaluationDomain as D, Evaluations as E};
+use crate::domains::EvaluationDomains;
+use crate::gate::{GateType, CurrOrNext, CircuitGate};
+
+#[derive(Clone, Copy, Debug)]
+pub struct LocalPosition {
```

Comment:
If we could have any public type / field documented with `///` it would be swell :D